core/tests/protractor/libraryPagesTour.js

```diff
@@ -58,12 +58,6 @@ describe('Library pages tour', function() {
     users.logout();
   });
 
-  it('visits the recent explorations page', function() {
-    visitLibraryPage();
-    element(by.css('.protractor-test-library-recently-published')).click();
-    expect(browser.getCurrentUrl()).toContain('library/recently_published');
```

Comment:
I removed this because after removing the recently published category from the library page the e2e test here fails as it checks for the recently published category(which is no longer present).

The page still exists and can be visited using the url 'library/recently_published'. I've modified this test to check whether the url exists or not.

Moreover other e2e tests (suggestions, explorationFeedback, LibraryPageTour) have the EXPLORATION_CATEGORY as 'Random' and further testing in these cases after exploration creation use the recently published section to access the exploration which fails these tests. So to make sure that the tests pass we need to change the category to any of the categories that already exist.
I've changed it to Algorithms.